skyve-war/src/main/java/modules/admin/CommunicationTemplate/CommunicationTemplate.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <document xmlns="http://www.skyve.org/xml/document" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="CommunicationTemplate" xsi:schemaLocation="http://www.skyve.org/xml/document ../../../schemas/document.xsd">
+    <documentation>Defines reusable email templates with HTML markup for consistent formatting and branding across bulk communications, supporting variable substitution within template content.</documentation>
```

Comment:
I think this should mention that they are _explicitly_ intended to be used to provide common template styling for `Communications` (i.e. they are useless on their own).